thrift-gen/zipkincore/ttypes.go

```diff
@@ -719,6 +720,7 @@ type Span struct {
 	Debug             bool                `thrift:"debug,9" json:"debug,omitempty"`
 	Timestamp         *int64              `thrift:"timestamp,10" json:"timestamp,omitempty"`
 	Duration          *int64              `thrift:"duration,11" json:"duration,omitempty"`
+	TraceIDHigh 	  *int64 			  `thrift:"trace_id_high,12" json:"trace_id_high,omitempty"`
```

Comment:
odd formatting. Did you use `make thrift` to generate this file?  The Travis build actually failed with `panic: runtime error: invalid memory address or nil pointer dereference` fro this file